Library/Homebrew/utils/array.rb

```diff
@@ -0,0 +1,53 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Utils
+  extend T::Sig
+
+  # Converts the array to a comma-separated sentence where the last element is
```

Comment:
Is this a direct import/copy from `active_support`? If so, what is the license there and is this compliant with doing so?

Reply:
`active_support` lives in [Rails](https://github.com/rails/rails), which uses the MIT [license](https://github.com/rails/rails/blob/main/MIT-LICENSE). I think this is fine (but IANAL).

Reply:
@dduugg If this has been copy-pasted from there at all: let's note the BSD license and where it's come from there.

I still wonder whether it's worth some sort attempt at bundler hacks to automatically keep this up to date based on the source? 🤔

Reply:
@MikeMcQuaid I've copied the rails license above the method at your request (as I understand it). I think this differs from the `ActiveSupport` version in crucial ways, in particular the lack of `locale` support, for a bundler hack to be feasible.

@dhh Since the Rails license attributes copyright to you directly, do you require/want copyright attribution for Homebrew to incorporate derivatives of `ActiveSupport` monkey-patches, such as `Array#to_sentence`?

Reply:
License granted to copy as you please for Homebrew. Big fan!

Reply:
Thanks @dhh! Similarly big fan of Rails over here.